src/Microsoft.Health.Dicom.Core/Extensions/DicomMediatorExtensions.cs

```diff
@@ -81,11 +81,11 @@ public static class DicomMediatorExtensions
     }
 
     public static Task<RetrieveRenderedResponse> RetrieveRenderedDicomInstanceAsync(
-        this IMediator mediator, string studyInstanceUid, string seriesInstanceUid, string sopInstanceUid, ResourceType resourceType, IReadOnlyCollection<AcceptHeader> acceptHeaders, CancellationToken cancellationToken, int frameNumber = 0)
+        this IMediator mediator, string studyInstanceUid, string seriesInstanceUid, string sopInstanceUid, ResourceType resourceType, IReadOnlyCollection<AcceptHeader> acceptHeaders, int quality, CancellationToken cancellationToken, int frameNumber = 0)
     {
         EnsureArg.IsNotNull(mediator, nameof(mediator));
         return mediator.Send(
-            new RetrieveRenderedRequest(studyInstanceUid, seriesInstanceUid, sopInstanceUid, resourceType, frameNumber, acceptHeaders),
+            new RetrieveRenderedRequest(studyInstanceUid, seriesInstanceUid, sopInstanceUid, resourceType, frameNumber, quality, acceptHeaders),
```

Comment:
So the default frame Number is 0 right. But according to spec, it starts at 1. So if the user specifies 1st frame, are we going to return the second frame?

```
9.5.1.2.1 Frame Number
frame-number = %s"frameNumber" "=" uint
This parameter specifies a single frame within a multi-frame image Instance, as defined in PS3.3 that shall be returned. Its name is "frameNumber" and its value shall be a positive integer (i.e., starts at 1 not 0).
```

Reply:
Fo dicom starts their frame count at 0 index https://github.com/fo-dicom/fo-dicom/blob/9832ab62de7a79a1a78a19438df69581079f91b9/FO-DICOM.Core/Imaging/DicomImage.cs#L211

Should I make default 1 and then subtract 1 when doing image rendering and specify in spec that frame numbering starts at 1?

Reply:
Actually even in our validate frame code we start frame counting at 0 - 
![image](https://user-images.githubusercontent.com/31964511/234617326-714f481f-815e-43e6-9396-f84a6027a1d9.png)

Reply:
updated as discussed offline